components/NewOrder/AnimatedMultistep.js

```diff
@@ -92,9 +74,9 @@ const AnimatedMultistep = ({ steps, navigation }) => {
         attachments.set("service_fkey", serviceOrder);
         await attachments.save();
       } catch (error) {
-        Alert.alert(
-          "The file either could not be read, or could not be saved to Back4app.",
-          error
+        setSnackbar(
+          true,
+          "The file either could not be read, or could not be saved to Back4app."
```

Comment:
Ovde bi makla Back4app - stavi nesto genericno. There was an error encountered while reading or saving the file. il nesto tako